### PR TITLE
Added babyAndkids page

### PR DESCRIPTION
**Pull request by Baba-Yaga-1**
#### Describe the changes you've made
I have added the babyAndkids page.

#### Describe if there is any unusual behaviour of your code(Write `NA` if there isn't)
Yes, I am not able to set the correct padding and margin in the footer section.

Preview of home page - 

apnidukaan

Preview of babyAndkids page -

footer1

#### Test plan (OPTIONAL)
A good test plan should give instructions that someone else can easily follow.
How someone can test your code?

#### Checklist

- [X] I have mentioned related issue number in PR description.
- [ X] I have added screenshots and related demo links of my changes..
- [ X] The title of my pull request is a short description of the requested changes.
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] Open Source Program names(OPTIONAL: If you participated in any open-source program then please mention the program name here)

### CODE REVIEW

**Comment by QAZIMAAZARSHAD** [comment]
Share website link.

In screenshot in doesn't look proper.

File: static/style.css [hunk 1]
```diff
@@ -240,6 +240,96 @@ marquee{
...
+  
+  
+}
+p{
+  text-align: center;
+  font-size: 20px;
+  color: purple;
+  padding: 10px 10px 10px 10px;
+  
+}
```
Comment (satyampgt4) [inline static/style.css:293]: change all tag you have added that is not class or id 
and add class or which is parent of it like

```
p{
  text-align: center;
  font-size: 20px;
  color: purple;
  padding: 10px 10px 10px 10px;

}
```
change this to 
```
.offer p{
  text-align: center;
  font-size: 20px;
  color: purple;
  padding: 10px 10px 10px 10px;

}
```
do same of all other

File: static/style.css [hunk 2]
```diff
@@ -240,6 +240,96 @@ marquee{
...
+
+  /* margin: auto;
+  text-align: center; */
+}
+h3{
+  margin-top: 40px;
+  color: rgb(6, 57, 99);
+  
+  
+}
```
Comment (satyampgt4) [inline static/style.css:286]: change all tag you have added that is not class or id 
and add class or which is parent of it like

```
h3{
  margin-top: 40px;
  color: rgb(6, 57, 99);


}
```


change this to 


```
.offer h3{
  margin-top: 40px;
  color: rgb(6, 57, 99);


}
```

File: static/style.css [hunk 3]
```diff
@@ -240,6 +240,96 @@ marquee{
...
+
+ .carousel-item img{
+  height: 500px;
+  width: 300px;
+   
+ }
+ 
+p{
+  padding: 20px 50px 20px 50px;
+}
```
Comment (satyampgt4) [inline static/style.css:261]: change all tag you have added that is not class or id 
and add class or which is parent of it like

```
p{
  padding: 20px 50px 20px 50px;
}
```


change this to 

```
.offer p{
  padding: 20px 50px 20px 50px;
}
```

**Review by satyampgt4 — CHANGES_REQUESTED** [review]
also add link of baby and kid in home page

**Comment by Baba-Yaga-1** [comment]
Hey, I have made the changes. Can you check it?

The footer looks okay from my side.

footer2

**Comment by QAZIMAAZARSHAD** [comment]
Share a link for review.

**Comment by Baba-Yaga-1** [comment]
> 
> 
> Share a link for review.

https://baba-yaga-1.github.io/DevIncept/babyAndkids.html

**Comment by QAZIMAAZARSHAD** [comment]
Navbar is still wrong.

Fix it,

**Comment by Baba-Yaga-1** [comment]
> Navbar is still wrong.
> 
> Fix it,

You mean the logo right ? What about the footer , is it correct?
I have updated this page in my own repository and shared the link, so that's why the logo is not appearing.

**Comment by QAZIMAAZARSHAD** [comment]
Yes logo and search bar button.

And why only the baby page is working in your link?

**Comment by Baba-Yaga-1** [comment]
> 
> 
> Yes logo and search bar button.
> 
> And why only the baby page is working in your link?

Because I have uploaded only the baby page in my own repository and shared the deployed site. That's why other things are not working.

**Comment by Baba-Yaga-1** [comment]
Previews of the website, when I view it from the local repo -

navbar
footer2

**Comment by QAZIMAAZARSHAD** [comment]
Compare your page with other pages.

The search button is larger on your page.

**Comment by QAZIMAAZARSHAD** [comment]
My cart link is also not present.

**Comment by Baba-Yaga-1** [comment]
The first half is of the baby page and the other half is of home page,

search

**Comment by QAZIMAAZARSHAD** [comment]
@satyampgt4 review this PR and help in resolving the issue.

**Comment by QAZIMAAZARSHAD** [comment]
image
Screenshot (104)

**Comment by Baba-Yaga-1** [comment]
> 
> 
> image
> Screenshot (104)

Okay, I will make the changes in my own repo and show it to you.

**Comment by Baba-Yaga-1** [comment]
I have added the changes in my repo.

https://baba-yaga-1.github.io/DevIncept/babyAndkids.html

**Comment by QAZIMAAZARSHAD** [comment]
> I have added the changes in my repo.
> 
> https://baba-yaga-1.github.io/DevIncept/babyAndkids.html

image
Search button!!!

**Comment by Baba-Yaga-1** [comment]
I am facing the same issue again, the search button appears to look this way when I view it from the local repo. Even I have copied the same css code and same html code for the navbar.

homepage1
babypage

**Comment by QAZIMAAZARSHAD** [comment]
I suggest close this PR.

Fetch upstream. Clone the project again. Do necessary changes, Then push the code and make PR.

File: babyAndkids.html [hunk 4]
```diff
@@ -0,0 +1,379 @@
...
+            <button class="carousel-control-next" type="button" data-bs-target="#carouselExampleControls" data-bs-slide="next">
+              <span class="carousel-control-next-icon" aria-hidden="true"></span>
+              <span class="visually-hidden">Next</span>
+            </button>
+          </div>
+          <br>
+        </header>
+   
+
+    <section class="offer">
```
Comment (satyampgt4) [inline babyAndkids.html:115]: first add a class babyandkids
update this to 

`<section class="offer babyandkids">`

File: static/style.css [hunk 5]
```diff
@@ -267,6 +267,96 @@ marquee {
     background-color: rgb(67, 67, 163);
 }
 
+/* Baby and kids section */
+
+.carousel{
```
Comment (satyampgt4) [inline static/style.css:272]: `.babyandkids .carousel{`

File: static/style.css [hunk 6]
```diff
@@ -267,6 +267,96 @@ marquee {
...
+    width:100%;
+    height:500px;
+    /* left: 400px; */
+  
+  }
+  
+   .carousel-item img{
```
Comment (satyampgt4) [inline static/style.css:279]: `.babyandkids .carousel-item img{`

File: static/style.css [hunk 7]
```diff
@@ -267,6 +267,96 @@ marquee {
...
+    height: 500px;
+    width: 300px;
+     
+   }
+   
+  .image p{
```
Comment (satyampgt4) [inline static/style.css:285]: babyandkids

File: static/style.css [hunk 8]
```diff
@@ -267,6 +267,96 @@ marquee {
...
+    padding: 20px 50px 20px 50px;
+  }
+  #products{
```
Comment (satyampgt4) [inline static/style.css:288]: `.babyandkids #products{`

File: static/style.css [hunk 9]
```diff
@@ -267,6 +267,96 @@ marquee {
...
+    padding: 20px 40px 80px 40px;
+  /*   margin: 0 40px 0 40px; */
+    
+    
+  }
+  
+  .image{
```
Comment (satyampgt4) [inline static/style.css:295]: `.babyandkids .image{`

File: static/style.css [hunk 10]
```diff
@@ -267,6 +267,96 @@ marquee {
...
+    margin-right: 20px;
+    position: relative;
+    display: inline-block;
+    box-shadow: 1rem 1rem 1rem -0.9rem rgb(81, 3, 145);
+    border-radius: 30px;
+  
+    /* margin: auto;
+    text-align: center; */
+  }
+  .image h3{
```
Comment (satyampgt4) [inline static/style.css:307]: `.babyandkids .image h3{`

File: static/style.css [hunk 11]
```diff
@@ -267,6 +267,96 @@ marquee {
...
+    margin-top: 20px;
+    color: rgb(6, 57, 99);
+    
+    
+  }
+  .image p{
```
Comment (satyampgt4) [inline static/style.css:313]: babyandkids

File: static/style.css [hunk 12]
```diff
@@ -267,6 +267,96 @@ marquee {
...
+    color: purple;
+    padding: 10px 10px 10px 10px;
+    
+  }
+  .price {
+    color: grey;
+    font-size: 28px;
+  }
+  
+  button {
```
Comment (satyampgt4) [inline static/style.css:325]: `.babyandkids button {`

File: static/style.css [hunk 13]
```diff
@@ -267,6 +267,96 @@ marquee {
...
+    padding: 12px;
+    color: white;
+    background-color: #000;
+    text-align: center;
+    cursor: pointer;
+    width: 100%;
+    font-size: 18px;
+  }
+  
+  button:hover {
```
Comment (satyampgt4) [inline static/style.css:337]: `.babyandkids button:hover {`

File: static/style.css [hunk 14]
```diff
@@ -267,6 +267,96 @@ marquee {
...
+    opacity: 0.7;
+  }
+  .color-field {
```
Comment (satyampgt4) [inline static/style.css:340]: `.babyandkids .color-field {`

File: static/style.css [hunk 15]
```diff
@@ -267,6 +267,96 @@ marquee {
...
+    
+  }
+  .image p{
+    text-align: center;
+    font-size: 20px;
+    color: purple;
+    padding: 10px 10px 10px 10px;
+    
+  }
+  .price {
```
Comment (satyampgt4) [inline static/style.css:320]: `.babyandkids .price {`

**Review by satyampgt4 — CHANGES_REQUESTED** [review]
please update these changes

File: static/style.css (hunk 15 above)
Comment (Baba-Yaga-1) [reply static/style.css]: I have applied the changes.
Can you check it now ?

babypage1

**Review by satyampgt4 — APPROVED** [review]
LGTM!

image 
----------------------------------------------
image

**Pull request merged**